docs/guide/using-python.md

```diff
@@ -20,13 +20,13 @@ Webots starts Python using the standard `python` command line.
 As a consequence, it executes the first `python` binary found in the current `PATH`.
 If you want to use a different version of Python, please install it if needed and configure your environment so that it becomes the default `python` version when called from the command line in a terminal.
 Alternatively, you can change the default Python command from the Webots Preferences in the General tab.
-If you set it for example to `python3.7` instead of `python`, this version of python will be used by default, if available from the command line.
-It is also possible to set a different version of Python for each robot controller by editing the `[python]` section of the `runtime.ini` file in each robot controller directory and setting the `COMMAND` value to `python3`, `python3.7` or `python2.7`, etc.
+If you set it for example to `python3.8` instead of `python`, this version of python will be used by default, if available from the command line.
```

Comment:
```suggestion
If you set it for example to `python3.8` instead of `python`, this version of Python will be used by default, if available from the command line.
```